src/Util/Api.php

```diff
@@ -147,7 +147,7 @@ public function getAboutPage(): string {
 		} );
 		// Rewrite some parts of the returned HTML.
 		$document = new DOMDocument( '1.0', 'UTF-8' );
-		$document->loadXML( $content );
+		$document->loadHTML( $content );
```

Comment:
I'm getting this epubcheck error with this change:

> ERROR(RSC-005): book.epub/OPS/about.xhtml(-1,-1): Error while parsing file: The processing instruction target matching "[xX][mM][lL]" is not allowed.

I think the XML prolog has to be stripped first.

Reply:
Thanks for finding this. I'll take a look

Reply:
The PageParser changes work fine for me without changing this line.

Reply:
> The PageParser changes work fine for me without changing this line.

I'm not sure what you mean.